README.md

```diff
@@ -5,7 +5,18 @@ Exercism Exercises in Elm
 
 ## Setup
 
-Detailed installation instructions can be found at [http://elm-lang.org/install](http://elm-lang.org/install).
+- Detailed installation instructions for elm can be found at [http://elm-lang.org/install](http://elm-lang.org/install).
+
+- Install `elm-test` for the command line with `npm install elm-test`.
+
+### Running Tests in an Exercise Directory
+Inside an exercise directory, Elm must be configured manually.
+
+- Install `laszlopandy/elm-console` and `deadfoxygrandpa/elm-test`.
+  - `elm-package install -y laszlopandy/elm-console`
+  - `elm-package install -y deadfoxygrandpa/elm-test`
+- You should then be able to run tests with `elm-test ExerciseModuleNameTest.elm`
+
```

Comment:
I've been running elm-test from the root of the repo, since it already had an `elm-package.json`. Since these requirements should be standard across all tests, might it make sense to have everyone do it that way rather then repeating these instructions for every test?